### PR TITLE
Update deprecated page to be useful

### PR DESCRIPTION
**Pull request by estelle**
The HTML `manifest` attribute is deprecated, but it's the first result when you search for manifest on MDN. This adds content to tell the user to not use appcache and provides links for what they were probably looking for when they landing on this page.

### CODE REVIEW

File: files/en-us/web/html/element/html/manifest/index.md [hunk 1]
```diff
@@ -12,15 +12,17 @@ browser-compat: html.elements.html.manifest
 
 The **`manifest`** attribute of the `<html>` element specifies a URL of an application cache manifest that is downloaded in the early stages of page load.
 
-> **Note:** manifest-based caching mechanism has been deprecated. Use service workers instead.
+> **Note:** manifest-based caching mechanism has been deprecated. Use [service workers](/en-US/docs/Web/API/Service_Worker_API) instead.
 
-The manifest attribute has only effect during early stages of page load, thus changing it via regular DOM interfaces has no effect, {{domxref("Window.applicationCache")}} interface instead.
+The `manifest` attribute only had an effect during early stages of page load, thus changing it via regular DOM interfaces had no effect. The deprecated {{domxref("Window.applicationCache")}} interface was used instead. The attribute functionality has been superceded by the [web app manifest](/en-US/docs/Web/Manifest), a JSON file deployed in from the {{CSSXref("head")}} of the HTML file with the {{CSSXref("link")}} with [`rel="manifest"`](/en-US/docs/Web/HTML/Attributes/rel) set.
```
Comment (Josh-Cena) [inline files/en-us/web/html/element/html/manifest/index.md:17]:
```suggestion
The `manifest` attribute only had an effect during early stages of page load, thus changing it via regular DOM interfaces had no effect. The deprecated {{domxref("Window.applicationCache")}} interface was used instead. The attribute functionality has been superceded by the [web app manifest](/en-US/docs/Web/Manifest), a JSON file deployed in from the {{HTMLElement("head")}} of the HTML file with the {{HTMLElement("link")}} with [`rel="manifest"`](/en-US/docs/Web/HTML/Attributes/rel) set.
```

Muscle memory? Same experience for me with `jsxref` ^^

Comment (estelle) [reply files/en-us/web/html/element/html/manifest/index.md:17]: Thanks. D'oh!

File: files/en-us/web/html/element/html/manifest/index.md [hunk 2]
```diff
@@ -12,15 +12,17 @@ browser-compat: html.elements.html.manifest
 
 The **`manifest`** attribute of the `<html>` element specifies a URL of an application cache manifest that is downloaded in the early stages of page load.
 
-> **Note:** manifest-based caching mechanism has been deprecated. Use service workers instead.
+> **Note:** manifest-based caching mechanism has been deprecated. Use [service workers](/en-US/docs/Web/API/Service_Worker_API) instead.
 
-The manifest attribute has only effect during early stages of page load, thus changing it via regular DOM interfaces has no effect, {{domxref("Window.applicationCache")}} interface instead.
+The `manifest` attribute only had an effect during early stages of page load, thus changing it via regular DOM interfaces had no effect. The deprecated {{domxref("Window.applicationCache")}} interface was used instead. The attribute functionality has been superceded by the [web app manifest](/en-US/docs/Web/Manifest), a JSON file deployed in from the {{HTMLElement("head")}} of the HTML file with the {{HTMLElement("link")}} with [`rel="manifest"`](/en-US/docs/Web/HTML/Attributes/rel) set.
```
Comment (Josh-Cena) [inline files/en-us/web/html/element/html/manifest/index.md:17]: I can't comment on the technical aspects of this, so I suggest asking for another review. I find this sentence a bit awkward to read, though:

```suggestion
The `manifest` attribute only had an effect during early stages of page load, thus changing it via regular DOM interfaces had no effect. The deprecated {{domxref("Window.applicationCache")}} interface was used instead. The attribute functionality has been superceded by the [web app manifest](/en-US/docs/Web/Manifest), an external JSON file associated with the HTML file through a {{HTMLElement("link")}} with [`rel="manifest"`](/en-US/docs/Web/HTML/Attributes/rel).
```

Also, is the attribute nonfunctional in other browsers, which is what the past tense implies?

**Review by Josh-Cena — APPROVED** [review]
I don't think someone else would come around to it soon and this appears reasonable to me, so I'm merging. Not to say I'm actually experienced enough to do technical reviews for PRs like this :)

**Pull request merged**